Gruntfile.js

```diff
@@ -82,12 +95,51 @@ module.exports = function (grunt) {
                     "sudo rm -f -r /var/lib/gpii"
                 ].join("&&")
             },
+            prepareRpmEnv: {
+                command: "dnf install -y nodejs-grunt-cli npm alsa-lib-devel json-glib-devel PackageKit-glib-devel libXrandr-devel libgnome-keyring-devel sudo @development-tools rpmdevtools"
+            },
+            buildRpmDocker: {
+                command: "docker run --rm -i -v $(pwd):/sync fedora /bin/bash -c 'dnf install -y nodejs-grunt-cli; cp -r /sync /packages; cd /packages; grunt buildrpm; cp -r /packages/bin /sync'"
```

Comment:
Was the [IDI docker-nodejs image](https://github.com/idi-ops/docker-nodejs) not used here because it's based on CentOS 7? Perhaps a Fedora image should be created instead? The IDI docker-nodejs image uses the IDI ansible-nodejs role which [installs grunt using npm](https://github.com/idi-ops/ansible-nodejs/blob/master/vars/RedHat.yml#L60-L61) by default. Installing grunt using a Fedora package will result in an environment that will be different than all other Linux environments used for GPII projects.